api/catalog/api/serializers/image_serializers.py

```diff
@@ -38,34 +44,33 @@ class ImageSearchRequestSerializer(
     # Ref: ingestion_server/ingestion_server/categorize.py#Category
     category = serializers.CharField(
         label="category",
-        help_text="A comma separated list of categories; available categories "
-        "include `illustration`, `photograph`, and "
-        "`digitized_artwork`.",
+        help_text=make_comma_separated_help_text(IMAGE_CATEGORIES, "categories"),
         required=False,
     )
     aspect_ratio = serializers.CharField(
         label="aspect_ratio",
-        help_text="A comma separated list of aspect ratios; available aspect "
-        "ratios include `tall`, `wide`, and `square`.",
+        help_text=make_comma_separated_help_text(ASPECT_RATIOS, "aspect ratios"),
         required=False,
     )
     size = serializers.CharField(
         label="size",
-        help_text="A comma separated list of image sizes; available sizes"
-        " include `small`, `medium`, or `large`.",
+        help_text=make_comma_separated_help_text(IMAGE_SIZES, "image sizes"),
         required=False,
     )
 
     @staticmethod
-    def validate_categories(value):
-        valid_categories = {"illustration", "digitized_artwork", "photograph"}
-        _validate_enum("category", valid_categories, value)
+    def validate_category(value):
+        _validate_enum("category", IMAGE_CATEGORIES, value)
         return value.lower()
 
     @staticmethod
     def validate_aspect_ratio(value):
-        valid_ratios = {"tall", "wide", "square"}
-        _validate_enum("aspect ratio", valid_ratios, value)
+        _validate_enum("aspect ratio", ASPECT_RATIOS, value)
+        return value.lower()
+
+    @staticmethod
+    def validate_size(value):
+        _validate_enum("size", IMAGE_SIZES, value)
```

Comment:
This validation wasn't happening before, so invalid `size` entries were permissible. This is now updated using the values defined in the help text.